javascript.js

```diff
@@ -0,0 +1,27 @@
+const myOpen = document.querySelector('.burger');
+const nav = document.querySelector('.header-menu');
+const myClose = document.querySelector('.close');
+const myLinks = document.querySelector('.links');
+
+myOpen.onclick = () => {
+  nav.classList.add('open');
+  myOpen.classList.add('display');
+};
+myClose.onclick = () => {
+  this.parentElement.classList.remove('open');
```

Comment:
Nice work on the popup 💯 but this line is throwing an error, seems like `parentElement` here is undefined, which is making it impossible to get the `classList`, which is also causing your menu not to close/disappear when the ❌ button is clicked.